provisioner/terraform/testdata/display-apps/display-apps.tf

```diff
@@ -0,0 +1,26 @@
+terraform {
+  required_providers {
+    coder = {
+      source  = "example.com/coder/coder"
```

Comment:
I'm not entirely sure what to do about this, I guess I can update it to the version that the plugin _will_ be?

Reply:
Yeah it's just that the feature isn't merged into the terraform provider yet [here](https://github.com/coder/terraform-provider-coder/pull/147). So I guess once I get approval here + there Ill merge the terraform one and release and update the `source` here prior to merging.

Reply:
SGTM!

Reply:
fixed